modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -133,10 +160,31 @@ panda  User
 Successfully deleted role "red"
 ----
 
+Redpanda Console::
+
+Administrators use Redpanda Console to delete an existing role by following these steps:
+
+1. Select the `Access Control` tab.
+
+2. Click the role you wish to delete. This will show all currently assigned permissions (ACLs) and principals (users).
+
+3. Click the `Delete` button.
+
+4. Redpanda Console shows you a prompt asking you to confirm deletion of the role. The prompt differs based on whether you have principals assigned to the role or not. If you have principals assigned to the role, you must enter the text `delete` in the input field where prompted before you may continue.
+
+5. Click the `Delete` button.
+
+====
```

Comment:
Not sure how the UI shook out, but should we mention that this will also delete all the ACLs associated with the role?

Reply:
The language is there above the steps for using console.

<img width="722" alt="image" src="https://github.com/redpanda-data/docs/assets/73869338/ffb98a91-55aa-489c-8f50-ab7444dae40f">